rubyUtils.gradle

```diff
@@ -25,7 +25,7 @@ buildscript {
     dependencies {
         classpath "org.yaml:snakeyaml:${snakeYamlVersion}"
         classpath "de.undercouch:gradle-download-task:4.0.4"
-        classpath "org.jruby:jruby-core:9.3.10.0"
+        classpath "org.jruby:jruby-complete:9.4.0.0"
```

Comment:
do we need to reintroduce the complete jar here?

Reply:
The version change was done before #15050 was merged, in that PR the `-complete` to `-core` was made: https://github.com/elastic/logstash/pull/15050/files#diff-7a2156b270dee259f243cf6371cdeb28e3bfdbc8c0e418cb37cfb4724144b305L28

Reply:
Fixed with 8dd850ff8ac87592128239ba2c1de045a7d47f02